tests/test_complex_types.cpp

```diff
@@ -9,6 +9,13 @@ using namespace sycl::ext::cplx;
       static_assert(                                                           \
           std::is_same_v<complex<T>, decltype(declval<complex<T>>() +          \
                                               declval<complex<T>>())>);        \
```

Comment:
I guess we should just remove this one.
It will also simplify the merge conflict with the previous PR :)

Reply:
I have removed it so it hopefully wont create a merge conflict. :crossed_fingers: